modules/web/server/src/main/scala/observe/web/server/http4s/ObserveUIApiRoutes.scala

```diff
@@ -195,15 +195,15 @@ class ObserveUIApiRoutes[F[_]: Async](
                              engineEvents(clientId)
                       ).parJoinUnbounded
                         .onFinalize[F](clientsDb.removeClient(clientId))
-          ws       <- WebSocketBuilder[F]
-                        .copy(filterPingPongs = false)
+          ws       <- wsBuilder
+                        .withFilterPingPongs(filterPingPongs = false)
```

Comment:
Is this on the backend? What does `FilterPingPong` is?

Reply:
No idea.